lib/commit-analyzer.js

```diff
@@ -0,0 +1,33 @@
+const { exec } = require('child_process');
+
+const cmd = (cmd, cb) => exec(cmd, { cwd: __dirname }, (err, stdout) => cb(stdout.split('\n').join('')));
+
+/**
+ * Analyzes all commits after last git tag and calls cb with one of
+ * "major", "minor", "patch" as a second parameter or
+ * ERROR_NO_CHANGES by default as first parameter.
+ * Looks for [major], [minor] in commit history.
+ *
+ * @param cb - Callback.
+ */
+module.exports = (cb) => {
+    let versionType;
```

Comment:
Please use 2 spaces as indentation. Not sure why linter doesn't complain about it.